bookkeeper-server/conf/bk_server.conf

```diff
@@ -157,22 +176,29 @@ explicitLacInterval = 1
 # Maximum writes to buffer to achieve grouping
 # journalBufferedWritesThreshold=524288
 
+# Maximum entries to buffer to impose on a journal write to achieve grouping.
+# journalBufferedEntriesThreshold=0
+
 # If we should flush the journal when journal queue is empty
 # journalFlushWhenQueueEmpty=false
 
 # The number of threads that should handle journal callbacks
 # numJournalCallbackThreads=1
 
+# The journal format version to write.
```

Comment:
add following:

    available formats are 1-5:
    1: no header
    2: a header section was added
    3: ledger key was introduced
    4: fencing key was introduced
    5: expanding header to 512 and padding writes to align sector size configured by `journalAlignmentSize`

    By default, it is `4`. If you'd like to enable `padding-writes` feature, you can set journal version to `5`.
    You can disable `padding-writes` by setting journal version back to `4`. This feature is available in 4.5.0
    and onward versions.